sqlflow/client.py

```diff
@@ -69,6 +69,9 @@ def sql_request(self, sql):
         user_id = os.getenv("SQLFLOW_USER_ID", "")
         hive_location = os.getenv("SQLFLOW_HIVE_LOCATION", "")
         hdfs_namenode_addr = os.getenv("SQLFLOW_HDFS_NAMENODE_ADDR", "")
+        # environment variables JUPYTER_HADOOP_USER, JUPYTER_HADOOP_PASS stores the user's hdfs credentials.
+        hdfs_user = os.getenv("JUPYTER_HADOOP_USER", "")
```

Comment:
Using the unified prefix `SQLFLOW_` ?

Reply:
The name `JUPYTER_HADOOP_USER` is used by didi currently.

Reply:
Should refine this later.

Reply:
I see..